nautobot/ipam/models.py

```diff
@@ -1440,6 +1469,17 @@ def get_vminterfaces(self):
         # Return all VM interfaces assigned to this VLAN
         return VMInterface.objects.filter(Q(untagged_vlan_id=self.pk) | Q(tagged_vlans=self.pk)).distinct()
 
+    def clean(self):
+        super().clean()
+
+        # Validate Vlan Group Range
+        if self.vlan_group:
+            _vlan_group_expanded_range = parse_numeric_range(self.vlan_group.range)
+            if self.vid not in _vlan_group_expanded_range:
```

Comment:
Might be nice to add a `contains(value)` method to `PositiveRangeNumberTextField` as a convenience helper?

Reply:
I am not sure if this can be done. `self.expanded_value` would be actually assigned (computed) during `to_python`